### PR TITLE
add Conflux Scaffold Documentation

### PR DESCRIPTION
**Pull request by intrepidcanadian**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING#create-a-pull-request).
- [x] **This is NOT a PR to submit translation**

- - -
This change is [Reviewable](https://reviewable.io/reviews/Conflux-Chain/conflux-documentation/425)

### CODE REVIEW

File: docs/espace/tutorials/scaffoldCfx/scaffold.md
Comment (darwintree) [inline docs/espace/tutorials/scaffoldCfx/scaffold.md:98]: This network configuration is defined in hardhat. I suggest the network definition would be better to be consistent with viem definition

```suggestion
yarn deploy --network confluxESpace
```

File: docs/espace/tutorials/scaffoldCfx/scaffold.md
Comment (darwintree) [inline docs/espace/tutorials/scaffoldCfx/scaffold.md:104]: This network configuration is defined in hardhat. I suggest the network definition would be better to be consistent with viem definition

```suggestion
yarn deploy --network confluxESpaceTest
```

File: docs/espace/tutorials/scaffoldCfx/scaffold.md
Comment (darwintree) [inline docs/espace/tutorials/scaffoldCfx/scaffold.md:112]:
```suggestion
targetNetworks: [chains.confluxESpace]
```

File: docs/espace/tutorials/scaffoldCfx/scaffold.md
Comment (darwintree) [inline docs/espace/tutorials/scaffoldCfx/scaffold.md:57]: This network configuration is defined in hardhat. I suggest the network definition would be better to be consistent with viem definition

```suggestion
yarn deploy --network confluxESpaceTest
```

**Review by darwintree — CHANGES_REQUESTED** [review]
Than you for your pull request. This is really an awesome pull request! However, some changes are required before merging.

1. Removing the lock file and `.yarn` cache files as well as `.yarnrc.yml`. These files are not needed for the new contents for document building.
2. Changing the network abbreviation to make it same as in viem definition (already suggested in document files but more changes are needed in scaffold project file)
3. Would you mind transferring the repo to conflux-fans? In that way we could maintain the repo for the following up security patches or requested new features.
4. This is not requested change but I think adding it would be better: how to change the configuration of an existed projects created from scaffold-eth to make it support conflux espace. This could serve as a individual issue and pull request.

File: package-lock.json
Comment (intrepidcanadian) [reply package-lock.json:1]: Removed

File: yarn.lock
Comment (intrepidcanadian) [reply yarn.lock:1]: Removed

File: docs/espace/tutorials/scaffoldCfx/scaffold.md
Comment (intrepidcanadian) [reply docs/espace/tutorials/scaffoldCfx/scaffold.md:23]: can you help with giving me permission to transfer? When I try to transfer to conflux-fans, I get the following message

"You don’t have the permission to create public repositories on conflux-fans"

**Comment by intrepidcanadian** [comment]
Thanks for reviewing and the suggestions! I think I am just missing moving over the repo to conflux-fans once you provide me permission to create a public repo on conflux-fans and then I can change the git clone SSH link in the instructions

Thanks!

**Comment by darwintree** [comment]
I've figured out a workaround that a repo is created from scratch and I've invited you as a maintainer: https://github.com/conflux-fans/conflux-scaffold.
Except for this change, I think you've misunderstood the repo to remove the build files. It seems build files are removed from conflux-scaffold (but should removed from **conflux-documentation**!). I think you need to do the following operation:

```
git rm -r .yarn
git rm package.lock .yarnrc.yml
```

and restoring `yarn.lock`

**Comment by intrepidcanadian** [comment]
> I've figured out a workaround that a repo is created from scratch and I've invited you as a maintainer: https://github.com/conflux-fans/conflux-scaffold. Except for this change, I think you've misunderstood the repo to remove the build files. It seems build files are removed from conflux-scaffold (but should removed from **conflux-documentation**!). I think you need to do the following operation:
> 
> ```
> git rm -r .yarn
> git rm package.lock .yarnrc.yml
> ```
> 
> and restoring `yarn.lock`

Okay! I think I have made the changes requested above. I have removed package-lock, yarmrc.yml and .yarn. Also have restored yarn.lock. I have also re-adjusted the url for the repos to reflect the url of the repo in conflux-fans

**Comment by darwintree** [comment]
> > I've figured out a workaround that a repo is created from scratch and I've invited you as a maintainer: https://github.com/conflux-fans/conflux-scaffold. Except for this change, I think you've misunderstood the repo to remove the build files. It seems build files are removed from conflux-scaffold (but should removed from **conflux-documentation**!). I think you need to do the following operation:
> > ```
> > git rm -r .yarn
> > git rm package.lock .yarnrc.yml
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > and restoring `yarn.lock`
> 
> Okay! I think I have made the changes requested above. I have removed package-lock, yarmrc.yml and .yarn. Also have restored yarn.lock. I have also re-adjusted the url for the repos to reflect the url of the repo in conflux-fans

From the commit history, It seems the yarn.lock is not restored in the latest 2 commits. Would you recheck it?

**Review by darwintree — APPROVED** [review]

**Pull request merged**
